crates/sdk/src/background_connection.rs

```diff
@@ -338,12 +336,24 @@ impl BackgroundDbConnection {
         Ok(())
     }
 
+    pub fn disconnect(&mut self) {
+        self.send_chan = None;
+        self.client_cache = None;
+        if let Some(h) = self.websocket_loop_handle.take() {
+            let _ = self.handle.block_on(h);
+        }
+        if let Some(h) = self.recv_handle.take() {
+            let _ = self.handle.block_on(h);
+        }
+        self.runtime = None;
```

Comment:
Oh, we also shouldn't be un-setting the runtime - it's required to still exist if we want to re-connect.

Reply:
hmm fair enough